drivers/hmis/app/models/hmis/hud/concerns/has_custom_data_elements.rb

```diff
@@ -8,6 +8,9 @@ module Hmis::Hud::Concerns::HasCustomDataElements
   extend ActiveSupport::Concern
 
   included do
+    # throwing this in here because if you have CDEs you are most likely submittable via a form- maybe move to its own concern
+    has_one :form_processor, class_name: 'Hmis::Form::FormProcessor', as: :owner, dependent: :destroy
+
```

Comment:
do one of
* remove comment
* change name of concern to something like `FormSubmittable` that encompasses both CDEs and form processor
* move to a new concern

Reply:
https://github.com/greenriver/hmis-warehouse/pull/4420/commits/850f43db90a4cbd23cbf2c4d70c0031129125939